boilerplate/app/i18n/i18n.ts

```diff
@@ -9,14 +9,19 @@ import ko from "./ko"
 import fr from "./fr"
 import jp from "./jp"
 
-i18n.fallbacks = true
+// Migration guide from i18n 3.x -> 4.x:
+// https://github.com/fnando/i18n-js/blob/main/MIGRATING_FROM_V3_TO_V4.md
+// https://github.com/fnando/i18n/discussions/24
 
 // to use regional locales use { "en-US": enUS } etc
-i18n.translations = { ar, en, "en-US": en, ko, fr, jp }
-
 const fallbackLocale = "en-US"
+export const i18n = new I18n(
+  { ar, en, "en-US": en, ko, fr, jp },
+  { locale: fallbackLocale, defaultLocale: fallbackLocale, enableFallback: true },
+)
```

Comment:
It looks like we also have some docs references to the old format:

<img width="954" alt="Screenshot 2024-08-13 at 9 20 50 AM" src="https://github.com/user-attachments/assets/ffb7ed6c-03de-48cd-8c77-3062f2096293">

Other parts of that doc `Internationalization in Ignite Apps` look outdated too (mention of a JSON file in i18n.translations, line 19-21 isn't what it references anymore)

Reply:
Updated with a link to the docs so it's up-to-date